geometry_msgs/Pose[] cam_poses
int64[] cam_ids
# 2-D pixel coordinates of the landmarks observed
geometry_msgs/Point[] landmarks_pixels
int64[] landmarks_ids
int64[] association_cam_ids
# association, id of the landmarks observed, and the cam id that observed it.

